subsys/caf/modules/sensor_sampler.c

```diff
@@ -302,9 +330,62 @@ static int sensor_trigger_init(const struct sensor_config *sc, struct sensor_dat
 		return -ENOMEM;
 	}
 
+	reset_sensor_sleep_cnt(sc, sd);
+
+	LOG_INF("Trigger configured (threshold: %d, time: %d ms)",
+		(int)(1000 * sc->trigger->activation.thresh),
+		sc->trigger->activation.timeout_ms);
 	return 0;
 }
 
+static void configure_max_power_state(void)
+{
+	if (IS_ENABLED(CONFIG_CAF_SENSOR_SAMPLER_ACTIVE_PM)) {
+		static enum power_manager_level last_power_state = POWER_MANAGER_LEVEL_MAX;
+		enum power_manager_level power_state;
+		bool active = false;
+		bool trigger_present = false;
+
+		/* Travel through all the sensors and check its state
+		 * and configuration.
+		 * Depending on the "most active" sensor the generic allowed
+		 * power state would be decided.
+		 */
+		for (size_t i = 0; i < ARRAY_SIZE(sensor_data); i++) {
+			struct sensor_data *sd = &sensor_data[i];
+			const struct sensor_config *sc = &sensor_configs[i];
+
+			if (atomic_get(&sd->state) == SENSOR_STATE_ERROR) {
+				/* Ignoring sensor in error state */
```

Comment:
Please finish sentences with dots. This is for consistency with other code we created so far.

Reply:
Ok